app/models/course.rb

```diff
@@ -0,0 +1,5 @@
+class Course < ApplicationRecord
+  has_many :reservations
```

Comment:
- [ ] In the frontend, if I try to delete a course that has a reservation:

### 1. "test course" has been reserved, then deleted:

![image](https://github.com/coderacheal/skill-habour-be/assets/57421823/43767591-c3f4-4633-bf1a-08e74985e538)

### 2. The reservation still exists.

![image](https://github.com/coderacheal/skill-habour-be/assets/57421823/d97c4e0a-eb5d-419f-8408-1b971c6f2fa6)

## Solution

In the Course model on line 2, add this:

```ruby
has_many :reservations, dependent: :destroy
```

That way, if a course ceases to exist, all its reservations will be deleted too. 💪